### PR TITLE
Update AudioLimiter.yaml

### PR DESCRIPTION
**Pull request by rbxphogen**
## Changes

Add a description for AudioLimiter

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 1]
```diff
@@ -2,17 +2,21 @@ name: AudioLimiter
 type: class
 category:
 memory_category: Internal
-summary: ''
-description: ''
+summary: |
+  `Class.AudioLimiter` limits how loud audio streams are allowed to be.
```
Comment (matthewzhang-rbx) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:6]:
```suggestion
  Limits how loud audio streams are allowed to be. 
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 2]
```diff
@@ -29,8 +33,10 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.MaxLevel
-    summary: ''
-    description: ''
+    summary: 'The maximum volume tolerated.'
+    description: |
+      The maximum volume, in decibels, that the limiter will allow to pass through, unclamped.
+      Whenever
```
Comment (matthewzhang-rbx) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:39]: It looks like you meant to write something here?

Comment (rbxphogen) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:39]: You're right lol; probably
```suggestion
      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel`, the output stream's volume will be reduced to compensate.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 3]
```diff
@@ -47,8 +53,9 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.Release
-    summary: ''
-    description: ''
+    summary: 'The amount of time it takes to unclamp previously-loud streams.'
+    description: |
+      The amount of time, in seconds, that it takes for the limiter to unclamp any previously (but not currently) loud streams.
```
Comment (matthewzhang-rbx) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:58]: We could add a sentence giving the range for this parameter

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 4]
```diff
@@ -29,8 +33,10 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.MaxLevel
-    summary: ''
-    description: ''
+    summary: 'The maximum volume tolerated.'
+    description: |
+      The maximum volume, in decibels, that the limiter will allow to pass through, unclamped.
+      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel`, the output stream's volume will be reduced to compensate.
```
Comment (matthewzhang-rbx) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:39]: We could add a sentence giving the range for this parameter

Comment (rbxphogen) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:39]: Oh good call
```suggestion
      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel`, the output stream's volume will be reduced to compensate.
      This value ranges from -12 to 0.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml (hunk 3 above)
Comment (rbxphogen) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:58]: Good call
```suggestion
      The amount of time, in seconds, that it takes for the limiter to unclamp any previously (but not currently) loud streams.
      This value ranges from 0.001 to 1.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 5]
```diff
@@ -29,8 +33,11 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.MaxLevel
-    summary: ''
-    description: ''
+    summary: 'The maximum volume tolerated.'
+    description: |
+      The maximum volume, in decibels, that the limiter will allow to pass through, unclamped.
```
[the remark this answers is not in the document]
Comment (rbxphogen) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:38]:
```suggestion
      The maximum volume, in decibels, that the limiter will allow to pass through, without reduction.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 6]
```diff
@@ -47,8 +54,10 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.Release
-    summary: ''
-    description: ''
+    summary: 'The amount of time it takes to unclamp previously-loud streams.'
```
[the remark this answers is not in the document]
Comment (rbxphogen) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:57]:
```suggestion
    summary: 'The amount of time it takes for previously-loud streams to return to their normal volume.'
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 7]
```diff
@@ -47,8 +54,10 @@ properties:
...
+    description: |
+      The amount of time, in seconds, that it takes for the limiter to unclamp any previously (but not currently) loud streams.
```
Comment (rbxphogen) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:59]:
```suggestion
      The amount of time, in seconds, that it takes for any previously (but not currently) loud streams to return to their normal volume.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 8]
```diff
@@ -2,17 +2,21 @@ name: AudioLimiter
...
 code_samples: []
 inherits:
   - Instance
 tags: []
 deprecation_message: ''
 properties:
   - name: AudioLimiter.Bypass
-    summary: ''
-    description: ''
+    summary: 'Whether audio streams are passed-through unaffected by this effect.'
```
[the remark this answers is not in the document]
Comment (rbxphogen) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:18]: This is copied from other descriptions for bypass, so we'd want to update those as well

[the remark this answers is not in the document]
Comment (matthewzhang-rbx) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:18]: I like "Whether audio streams pass through this effect without change" - we could update all of the Bypass descriptions in a follow-up

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 9]
```diff
@@ -47,8 +54,10 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.Release
-    summary: ''
-    description: ''
+    summary: 'The amount of time it takes for previously-loud streams to return to their normal volume.'
+    description: |
+      The amount of time, in seconds, that it takes for any previously (but not currently) loud streams to return to their normal volume.
```
Comment (matthewzhang-rbx) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:59]:
```suggestion
      The amount of time, in seconds, that it takes for any previously (but not currently) limited streams to return to their normal volume.
```

**Review by matthewzhang-rbx — DISMISSED** [review]
lgtm

File: content/en-us/reference/engine/classes/AudioLimiter.yaml (hunk 8 above)
[the remark this answers is not in the document]
Comment (fumminger) [reply content/en-us/reference/engine/classes/AudioLimiter.yaml:18]: Yes, we should update them all to use active voice.

**Review by fumminger — APPROVED** [review]

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 10]
```diff
@@ -2,17 +2,21 @@ name: AudioLimiter
...
 category:
 memory_category: Internal
-summary: ''
-description: ''
+summary: |
+  Limits how loud audio streams are allowed to be. 
+description: |
+  `Class.AudioLimiter` limits how loud audio streams are allowed to be.
+  Whenever its input stream exceeds a specified maximum level, the stream's volume is reduced for a moment.
+  `Class.AudioLimiter` provides a single **Input** pin, and a single **Output** pin that can be connected to/from by `Class.Wires`.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:10]:
```suggestion
  `AudioLimiter` limits how loud audio streams are allowed to be.
  Whenever its input stream exceeds a specified maximum level, the stream's volume is reduced for a moment.
  `AudioLimiter` provides a single **Input** pin, and a single **Output** pin that can be connected to/from by `Class.Wires`.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml (hunk 8 above)
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:18]:
```suggestion
    summary: |
      Whether audio streams are passed-through unaffected by this effect.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 11]
```diff
@@ -2,17 +2,21 @@ name: AudioLimiter
...
 inherits:
   - Instance
 tags: []
 deprecation_message: ''
 properties:
   - name: AudioLimiter.Bypass
-    summary: ''
-    description: ''
+    summary: 'Whether audio streams are passed-through unaffected by this effect.'
+    description: 'If `true`, audio streams are passed-through unaffected by this effect.'
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:19]:
```suggestion
    description: |
      If `true`, audio streams are passed-through unaffected by this effect.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 12]
```diff
@@ -29,8 +33,11 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.MaxLevel
-    summary: ''
-    description: ''
+    summary: 'The maximum volume tolerated.'
+    description: |
+      The maximum volume, in decibels, that the limiter will allow to pass through, without reduction.
+      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel`, the output stream's volume will be reduced to compensate.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:39]:
```suggestion
      The maximum volume, in decibels, that the limiter will allow to pass through without reduction.
      Whenever the input stream exceeds `Class.AudioLimiter.MaxLevel|MaxLevel`, the output
      stream's volume will be reduced to compensate.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 13]
```diff
@@ -29,8 +33,11 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.MaxLevel
-    summary: ''
-    description: ''
+    summary: 'The maximum volume tolerated.'
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:36]:
```suggestion
    summary: |
      The maximum volume tolerated.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 14]
```diff
@@ -47,8 +54,10 @@ properties:
       - Audio
     writeCapabilities: []
   - name: AudioLimiter.Release
-    summary: ''
-    description: ''
+    summary: 'The amount of time it takes for previously-loud streams to return to their normal volume.'
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:57]:
```suggestion
    summary: |
      The amount of time it takes for previously-loud streams to return to their normal volume.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 15]
```diff
@@ -47,8 +54,10 @@ properties:
...
+    description: |
+      The amount of time, in seconds, that it takes for any previously (but not currently) limited streams to return to their normal volume.
+      This value ranges from 0.001 to 1.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:60]:
```suggestion
      The amount of time, in seconds, that it takes for any previously (but not currently) limited streams to
      return to their normal volume. This value ranges from `0.001` to `1`.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 16]
```diff
@@ -66,8 +75,9 @@ properties:
     writeCapabilities: []
 methods:
   - name: AudioLimiter:GetConnectedWires
-    summary: ''
-    description: ''
+    summary: 'Returns an array of `Class.Wires` that are connected to the specified pin.'
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:78]:
```suggestion
    summary: |
      Returns an array of `Class.Wire|Wires` that are connected to the specified pin.
```

File: content/en-us/reference/engine/classes/AudioLimiter.yaml [hunk 17]
```diff
@@ -66,8 +75,9 @@ properties:
...
+    description: |
+      Returns an array of `Class.Wires` that are connected to the specified pin. `Class.AudioLimiter` has one "Input" pin, and one "Output" pin.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/AudioLimiter.yaml:80]:
```suggestion
      Returns an array of `Class.Wire|Wires` that are connected to the specified pin. `Class.AudioLimiter`
      has one **Input** pin and one **Output** pin.
```

**Review by IgnisRBX — APPROVED** [review]

**Pull request merged**